ehrapy/api/anndata_ext.py

```diff
@@ -32,13 +33,18 @@ def df_to_anndata(
     # if data is numerical only, short-circuit AnnData creation to have float dtype instead of object
     all_num = all(np.issubdtype(column_dtype, np.number) for column_dtype in dataframes.df.dtypes)
     X = dataframes.df.to_numpy(copy=True)
-
+    # initializing an OrderedDict with a non-empty dict might not be intended,
+    # see: https://stackoverflow.com/questions/25480089/right-way-to-initialize-an-ordereddict-using-its-constructor-such-that-it-retain/25480206
```

Comment:
Should we investigate this at some point and add a TODO here or is this fine?

Reply:
Will check this one out now. Just a few minutes.

Reply:
> "The OrderedDict constructor and update() method both accept keyword arguments, but their order is lost because Python’s function call semantics pass-in keyword arguments using a regular unordered dictionary"